src/components/ViewIssue/blocks/SubItems.tsx

```diff
@@ -8,19 +6,29 @@ import type { Issue, IssueSubItem } from "../../../services/space/types";
 
 export type Props = {
   subItems?: Issue["subItemsList"],
+  onCompleteItem: (
+    listId: Issue["subItemsList"]["id"],
+    itemId: IssueSubItem["id"],
+    resolved: boolean,
+  ) => Promise<unknown>,
```

Comment:
Can we type this to something other than unknown?

Reply:
changed as `Promise<void>`